src/legacy/core_plugins/vis_type_vislib/public/vislib/components/legend/legend.tsx

```diff
@@ -22,14 +22,24 @@ import { compact, uniq, map } from 'lodash';
 
 import { i18n } from '@kbn/i18n';
 import { EuiPopoverProps, EuiIcon, keyCodes, htmlIdGenerator } from '@elastic/eui';
+import { AggConfig } from 'ui/agg_types';
 
 // @ts-ignore
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { createFiltersFromEvent } from '../../../../../data/public/actions/filters/create_filters_from_event';
 import { CUSTOM_LEGEND_VIS_TYPES, LegendItem } from './models';
 import { VisLegendItem } from './legend_item';
 import { getPieNames } from './pie_utils';
-import { getTableAggs } from '../../../legacy_imports';
+
+import { Vis } from '../../../../../visualizations/public';
+import { tabifyGetColumns } from '../../../../../../ui/public/agg_response/tabify/_get_columns';
+
+const getTableAggs = (vis: Vis): AggConfig[] => {
```

Comment:
nit: I'm adding `IAggConfig` in #56353, so if that merges first, it would be preferable to use it here instead.